src/qtiCommonRenderer/renderers/interactions/TextEntryInteraction.js

```diff
@@ -128,6 +128,8 @@ var render = function render(interaction) {
             });
     } else if (attributes.patternMask) {
         updatePatternMaskTooltip = function updatePatternMaskTooltip() {
+            if (!$input.val().length) return;
```

Comment:
You cannot add code before the variable declaration. This is violating our ES5 coding style: reflect hoisting.

You should either move the code after the var declaration, or convert the code in ES6.